src/components/InfoBox/index.js

```diff
@@ -1 +1 @@
-export { default } from "./InfoBox";
+export { default, SUCCESS, ERROR, WARNING, ACTION_REQUIRED } from "./InfoBox";
```

Comment:
**TLDR;** 

Let's move this constants to default export so that we can use them as we used before. 

**Longer version**

It would be nice if we could attach these constants directly to the default export. Ideally using it as `InfoBox.ERROR` saves us from an additional import that each time we need to specify the type (and we specify pretty often). Also the constants `SUCCESS`, `ERROR` etc... are very generic names, so in case we have these constants already defined in the imported file it'll create additional issues.

Reply:
ok, I've commited the new version :)